docs/docs/glamor.md

```diff
@@ -19,7 +19,7 @@ First, open a new terminal window and run the following to create a new site:
 gatsby new glamor-tutorial https://github.com/gatsbyjs/gatsby-starter-hello-world
 ```
 
-Second, install the Gatsby plugin for Glamor.
+Second, we’ll install the necessary dependencies for Glamor and Gatsby.
```

Comment:
Remove `we'll` here, since the docs are intended for the person reading. You can learn more about this in the Gatsby style guide, which recommends using the more personal "you" for both the docs and tutorial (or in this case, omitting it): https://www.gatsbyjs.org/docs/gatsby-style-guide/#use-you-as-the-pronoun

I can see there are other uses of "we'll" up above in this doc; if you feel like rewording them to fit in with the style guide that would be awesome...but it would be totally fine to limit your changes to this line only.

Reply:
Thank you so much @marcysutton for pointing me to the style guide. I have made the suggested changes!